Documentation/ApiOverview/Hooks/Index.rst

```diff
@@ -23,3 +23,4 @@ its extensions without blocking others to do the same.
    Events/Index
    SignalsSlots/Index
    Hooks/Index
+   JavaScript/Index
```

Comment:
Would it make more sense to add it to https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/ApiOverview/JavaScript/Index.html?

The "Events, Signals and Hooks" is more PSR-14 and stuff like that, used in Frontend and Backend. This new chapter is only relevant in TYPO3 Backend where JavaScript is used? Or can one use it in Frontend as well?

We could also add a hint from the other chapter.

Reply:
The API is designed to be used in frontend as well, that's why it's placed in EXT:core. I have no strong opinion where this chapter is placed, but to me, it doesn't feel right, too.